### ISSUE TITLE
Experience Page Improvements  - Upload

### ISSUE THREAD

**Issue by jiraslama** [alan-turing-institute/AutisticaCitizenScience#353]
As a Fujitsu developer
I would like
A wireframe design and justification that showcases a improvement to the experience page with the further explanation for the user about what the Upload feature is. State that we have a upload section - Upload Modular Panel Explaining what its function is and that the user has a number of options: 
Option 1: Use another term instead of Upload - Submit Media (Present User Options)
Media Messaging Submit Media 
Options
1. Upload by image only
2. Upload by text
3. Upload by audio
Make the upload functionality more of a integral part of the experience page form - more within the input form. 
Because...
I would like show to the wider community a Fujitsu design concept for the platform.
Acceptance criteria
Axure updated with the final UI design for the component A picture/screenshot image of the final UI design of the component held within Axure Justification of the component design available in Github repository as an md file

**Comment by katoss** [comment]
**Media upload**
**The label “upload” is confusing:**

- P i: “What is important to remember about autistic people, is that they question everything. You think it is obvious? It is not. They will give you ten different explanations on how they understood it.”
- A (1) d: “I think once you entered the event, you can upload it to the database.”
- A (1) d: “If you would have an upload-section, I would rephrase it to something like ‘Upload by image only’ or ‘Upload by text only’ or ‘Upload by audio’.”
- A (2) d: “I don’t know what ‘upload’ means. I think I have to upload in the end. I don’t know how ‘upload’ is different than ‘submit’. ‘Submit’ speaks for itself, so ‘click it in the end when you are done’, but I don’t know why there is an upload there.”
- A (2) d: “The upload images-function should be more within it [the input form]. Because when it is within it, it implies ‘add more specificity’ to what you have written. When it is outside, it looks like you are changing the accessibility settings of this page.”

**Possibility to upload media in general**

- d: 2x appreciate (Other participants did not thematise that question directly)
- A (1) d:“I think pictures can be useful. I think somebody mentioned it to me before, so it doesn’t come as a surprise. It takes you back, but in a good way and could help other users to engage.

**Video upload**

- A D d: “I would not like to upload a video of myself. Maybe just me talking, but not a video of my face.”
- A C d: “As long as I am not in it and I am just showing the environment, possibly I could try. [...] Whereas when I am writing, I’m trying to take someone through a story. When I am doing a video, I tend to get over-excitable and say all sorts of stuff that might not be entirely useful. So, I don’t know. Other people maybe shoot a video and never say a thing.”

**→ Tendentially media upload is seen as positive, but not essential (except for voice-to-text). Towards the idea of uploading videos, the participants who commented on it were rather skeptical. The ‘upload’ button should be relabelled and possibly be placed more within the form.**

**Comment by jiraslama** [comment]
Options covered within wireframes

Upload by image only X
Upload by text     X
Upload by audio X

**Comment by jiraslama** [comment]
https://5wwsja.axshare.com/#g=1&p=upload click on upload button to view interactive wireframe

**Comment by jiraslama** [comment]
Updated wireframe https://5wwsja.axshare.com/#g=1&p=1-home-nav-access-settings
